includes/class-sensei-grading-main.php

```diff
@@ -425,7 +441,8 @@ public function data_table_header() {
 			foreach ( $menu as $class => $item ) {
 				$menu[ $class ] = "\t<li class='$class'>$item";
 			}
-			echo implode( " |</li>\n", $menu ) . "</li>\n";
+
+			echo implode( " |</li>\n", wp_kses_post( $menu ) ) . "</li>\n";
```

Comment:
`$menu` is not a string.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2296/commits/ce62481272d680fdc6e06a27edbdbe48efa00cc3.